tasks/main.yml

```
@@ -173,8 +173,8 @@
     {% if not odoo_role_demo_data %}
     --without-demo=all
     {% endif %}
-    --logfile=/dev/stdout
-    --log-level=warn
+    --logfile /dev/stdout
```

Undo substituting equal with space. If we want to do it, we should do that everywhere not only in a corner

```suggestion
    --logfile=/dev/stdout
```